cloudformation/launchdetector.yaml

```diff
@@ -16,6 +16,20 @@ Parameters:
       - preview
       - live
     Description: Which CAPI mode to listen to
+  PlutoHost:
+    Type: String
+    Description: Host or loadbalancer where pluto can be reached
+  PlutoVidispinePort:
+    Type: Number
+    Description: Port on which Vidispine can be reached for Pluto
+    Default: 8080
+  PlutoUser:
+    Type: String
+    Description: Username to use to communicate
+  PlutoPass:
```

Comment:
can we store credentials in S3 or dynamo rather than cfn params?

Reply:
sure, what is the reason?